docs/source/DeveloperGuide/AddingNewComponents/DevelopingDatabases/index.rst

```diff
@@ -36,7 +36,7 @@ The *DB* class includes all the methods necessary to access, query, modify... th
 The first line in the ``__init__`` method should be the initialization of the parent (*DB*) class. That initialization requires 2 or 3 parameters:
 
  1. Logging name of the database. This name will be used in all the logging messages generated by this class.
- 2. Full name of the database. With *System/Name*. So it can know where in the CS look for the initialization parameters. In this case it would be */Systems/Test/<instance name>/Databases/AtomDB*.
+ 2. Full name of the database. With *System/Name*. So it can know where in the CS look for the initialization parameters. In this case it would be */Systems/<instance name>/Databases/AtomDB*.
```

Comment:
```suggestion
 2. Full name of the database. With *System/Name*. So it can know where in the CS look for the initialization parameters. In this case it would be */Systems/Test/Databases/AtomDB*.
```